exec/java-exec/src/main/resources/bootstrap-storage-plugins.json

```diff
@@ -32,9 +32,9 @@
           delimiter: "\t"
         },
         "httpd" : {
-          type: "httpd",
-          logFormat: "%h %t \"%r\" %>s %b \"%{Referer}i\""
-          /* timestampFormat: "dd/MMM/yyyy:HH:mm:ss ZZ" */
+          "type": "httpd",
```

Comment:
Thank you. According to [JSON standard](https://tools.ietf.org/html/rfc8259#section-4) it is needed to wrap field names with quotes (`"`). Could you update the whole file, please?

Reply:
Fixed